### DISCUSSION TITLE
Add [pass sync test] label about pull request

### DISCUSSION THREAD

**Discussion by kslee8282** [aergoio/aergo#285] [General]
currently, some pr merged without sync test unintentionally.
it need to add a [pass sync test] label and get confirmation from QA (kslee8282) before merge.

**Comment by gosunuts** [comment]
i agree. all branch must be pass sync test before release.
but some pr ( about refactoring, testing, ans so on ) don`t need pass sync test before pull request.

**Reply by gosunuts** [reply]
i think some pr ( about refactoring, testing ) can add sync test label after merge, but it must add label before release.

**Reply by kroggen** [reply]
your sentence is not understandable

**Reply by kroggen** [reply]
PRs that only affect vm_dummy and integration tests can be merged without any sync test.
There is one open right now

**Comment by kroggen** [comment]
The label `pass sync test` was created
